cgroup.go

```diff
@@ -193,6 +196,31 @@ func (c *cgroup) AddTask(process Process, subsystems ...Name) error {
 	return c.add(process, cgroupTasks, subsystems...)
 }
 
+// writeCgroupsProcs writes to the file, but retries on EINVAL.
+func writeCgroupProcs(path string, content []byte, perm fs.FileMode) error {
+	f, err := os.OpenFile(path, os.O_CREATE|os.O_WRONLY, perm)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	for i := 0; i < 5; i++ {
```

Comment:
With how this is structured we'd retry even if the error we got wasn't EINVAL, right (just with no delay)? Should we just return the error directly if != EINVAL? The continue shouldn't be needed if we want to keep this how it is

Reply:
I guess this kind of plays into my [other comment](https://github.com/containerd/cgroups/pull/250/files#r1014478984)

Reply:
Oh right. Will fix.